www/src/components/Container/Container.js

```diff
@@ -34,6 +34,12 @@ const Container = ({children}) => (
       [media.size('xxlarge')]: {
         maxWidth: 1260,
       },
+
+      [media.size('sidebarFixedNarrowFooter')]: {
+        maxWidth: isFooter ? 800 : 1260,
+        paddingLeft: isFooter ? 0 : 20,
+        paddingRight: isFooter ? 0 : 20,
+      },
```

Comment:
Maybe rather than passing in `isFooter` we could support additional custom CSS?

Reply:
Good shout, feels nicer to compose. Changed :)